ultrack/core/export/ctc.py

```diff
@@ -371,8 +371,8 @@ def condition(node: Node) -> bool:
         )
 
     df["track_id"], fw, _ = relabel_sequential(df["track_id"].values)
-    fw[NO_PARENT] = NO_PARENT
-    df["parent_track_id"] = fw[df["parent_track_id"].values]
+    mask = df["parent_track_id"] != -1
```

Comment:
```suggestion
    mask = df["parent_track_id"] != NO_PARENT
```
I would rather use the variable `NO_PARENT` in case we change this in the future.